### PR TITLE
all p0 commits with carbon frames

### PR DESCRIPTION
**Pull request by mmanjuprasad**
I have built all the components priorly and this is the UI i am building on top of it
Please reveiw and provide a feedback
Thank you

### CODE REVIEW

**Comment by mmanjuprasad** [comment]
https://boring-mestorf-774e89.netlify.app/

Deploy websit

**Comment by kgajriaAgarwal** [comment]
image
margin padding required

image
sidebar and carbon iframes are overlapping

image
header and carbon iframes are overlapping

image
ui not proper

try to make it responsive
try to improve the ui for the documentation site as it is not looking up to the mark!

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,932 @@
...
+  }
+
+.nav-header-DW {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 0;
+    background: var(--secondary-color);
+    width: 100%;
+    min-width: 300px;
```
Comment (kgajriaAgarwal) [inline styles.css:23]: use rem

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,932 @@
...
+    display: inline-block;
+  }
+  .header__nav-DW a {
+    padding: 8px;
+  }
+  
+  
+  a {
+    text-decoration: none;
+    color: rgb(8, 8, 8);
```
Comment (kgajriaAgarwal) [inline styles.css:58]: mention color in variables

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,932 @@
...
+}
+
+.section p span{
+    background-color: var(--span-bg-color);
+    border-radius: 5px;
+    padding: 0.1rem;
+}
+
+.cui-text{
+    text-align: center !important;
```
Comment (kgajriaAgarwal) [inline styles.css]: avoid using important

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,932 @@
...
+    margin: 0;
+  }
+  
+  
+  .flex{
+      display: flex;
+  }
+  
+  .container{
+      gap: 30px;
```
Comment (kgajriaAgarwal) [inline styles.css:199]: use rem

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,932 @@
...
+  
+  .header img{
+      width: 100%;
+  }
+  .card-content{
+      flex-direction: column;
+  }
+  
+  .card-content:hover{
+      background-color: #f8f8f8;
```
Comment (kgajriaAgarwal) [inline styles.css:222]: use variable for color

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,932 @@
...
+  }
+  
+  .footer{
+      justify-content: space-between;
+      padding: 1rem 1.5rem;
+      align-items: center;
+  }
+  
+  .footer button{
+      color: #0346fcbd;
```
Comment (kgajriaAgarwal) [inline styles.css:249]: use variable for color

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,932 @@
...
+      height: 30px;
+      width: 30px;
+      border-radius: 50%;
+      font-weight: normal;
+      
+    }
+  
+    .header-standard {
+      display: flex;
+      background-color: #6200ee;
```
Comment (kgajriaAgarwal) [inline styles.css:278]: use variable for color

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,932 @@
...
+      display: flex;
+      /* flex-basis: 70%; */
+      /* overflow: hidden; */
+  
+    }
+    
+    .card-hz .card-header-img {
+      object-fit: cover;
+      flex-basis: 50%;
+      width: 80px;
```
Comment (kgajriaAgarwal) [inline styles.css:305]: use rem

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,932 @@
...
+
+  .image-fit{
+    object-fit: cover;
+}
+  /* Badge styling starts here */
+
+  .badge{
+    height: 2rem;
+    width: 2rem;
+    color:#f8f8f8;
```
Comment (kgajriaAgarwal) [inline styles.css:332]: use variable for color

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,932 @@
...
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+  }
+  
+  .bwbl button{
+    border: white;
+    background-color: #6767703f;
```
Comment (kgajriaAgarwal) [inline styles.css:343]: use variable for color

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,932 @@
...
+  min-height: 2rem;
+  border-radius: 3px;
+  padding: 0.75rem 1.25rem;
+  border: 1px solid transparent;
+  width: 60%;
+  flex-wrap: wrap;
+  margin: 1rem;
+}
+.alert-primary {
+  background-color: rgb(63, 127, 199);
```
Comment (kgajriaAgarwal) [inline styles.css:439]: use variable for color

File: styles.css [hunk 12]
```diff
@@ -0,0 +1,932 @@
...
+
+.snackbar .snackbar-action .snackbar-action-icon {
+  color: white;
+}
+
+
+/* Text utilities starts here  */
+
+.text-center{
+  text-align: center !important;
```
Comment (kgajriaAgarwal) [inline styles.css]: avoid important

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,932 @@
...
+}
+.text-left{
+  text-align: left !important;
```
Comment (kgajriaAgarwal) [inline styles.css]: avoid important

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,932 @@
...
+}
+.text-right{
+  text-align: right !important;
```
Comment (kgajriaAgarwal) [inline styles.css]: avoid important

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,932 @@
...
+.h6 {
+  font-weight: 500;
+  font-size: 1rem;
+}
+
+.grey-color{
+  color: grey
+}
+.blue-color{
+  color: #0004ff!important;
```
Comment (kgajriaAgarwal) [inline styles.css]: avoid important

**Review by kgajriaAgarwal — CHANGES_REQUESTED** [review]
there are alot of changes required all over your code and even ui also needs to be improved

File: styles.css [hunk 16]
```diff
@@ -0,0 +1,932 @@
...
+    min-width: 300px;
+    height: 3rem;
+    /* position: fixed; */
+
+  }
+
+  .fixed{
+      margin : none;
+      position: fixed;
+      /* padding-bottom: 3rem; */
```
Comment (Sahil25061999) [inline styles.css:32]: remove comments if not necessary

File: styles.css [hunk 17]
```diff
@@ -0,0 +1,932 @@
...
+  }
+  
+  .logo {
+    font-size: 20px;
```
Comment (Sahil25061999) [inline styles.css:36]: use rem

**Review by Sahil25061999 — COMMENTED** [review]
The padding for the sidebar can be increased and the z-index for the navbar can be increased.

**Pull request merged**